SPECS/nbd/nbd.spec

```diff
@@ -25,7 +26,7 @@ remote block devices over a TCP/IP network.
 %autosetup -p1
 
 %build
-%configure --enable-syslog --enable-lfs --enable-gznbd
+%configure --enable-syslog --enable-lfs
```

Comment:
🥇 or disabling this :)

Reply:
Good point! I checked their [nbd](https://github.com/NetworkBlockDevice/nbd/tree/master)
/configure.ac file, seems like there is only `--enable-gznbd` flag for toggling this option.